source/help/getting-started/creating-teams.rst

```diff
@@ -0,0 +1,44 @@
+Creating Teams
+==============
+
+New teams can be created if the System Admin has **Enable Team Creation** set to true from the System Console.
+
+Methods to Create a Team
+------------------------
+
+Teams can be created from the Team Selection page or Main Menu. The person who creates a team will automatically be given the Team Admin role for that team.
+
+Team Selection Page
+~~~~~~~~~~~~~~~~~~~~
+Navigate to the web address of your system, ``https://domain.com/``. Sign in using your credentials. Click **Create a new team** to be guided through the rest of the set up steps. If this option is not visible on the web page, then the System Admin has disabled team creation. It is necessary to have an existing account on the system in order to create a team from the system home page, so if you do not have an account you will need to create one.
```

Comment:
Break-up into smaller sentences? You must have an existing account on the system in order to create a team from the system home page. So, if you do not have an account, you will need to create one.

Or, if we don't want "you":
An existing account is required to create a team from the system home page. If the user doesn't have an account, they'll need to create one.